packages/qdrant-loader/src/qdrant_loader/connectors/confluence/connector.py

```
@@ -792,162 +804,249 @@ def _clean_html(self, html: str) -> str:
-                            f"Reached end of results: {start + limit} >= {total_size}"
-                        )
-                        break
-
-                    # Move to next page
-                    start += limit
-                    logger.debug(f"Moving to next page with start={start}")
+                                logger.debug(
+                                    f"Processed {content['type']} '{content['title']}' "
+                                    f"(ID: {content['id']}) from space {self.config.space_key}"
+                                )
+                        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)